lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/StakePools.hs

```diff
@@ -477,33 +497,6 @@ spec = do
         let poolId = toText $ getApiT $ p ^. #id
         expectErrorMessage (errMsg403PoolAlreadyJoined poolId) r
 
-    it "STAKE_POOLS_JOIN_01 - \
```

Comment:
I delete this as it is replicated (and even more) in 
```
it "STAKE_POOLS_JOIN/QUIT - Checking delegation expectations
```
added below

Reply:
In that case, the `HTTP 202` expectations should be moved too, I believe.

Reply:
they are corresponding : 
```
expectResponseCode HTTP.status202 rX
```
in the test that replaces this one